lib/generators/solid_cache/install/install_generator.rb

```diff
@@ -7,7 +7,7 @@ class SolidCache::InstallGenerator < Rails::Generators::Base
                                     desc: "Skip migrations"
 
   def add_rails_cache
-    if (env_config = Pathname(destination_root).join("config/environments/production.rb")).exist?
+    if (env_config = app_root.join("config/environments/production.rb")).exist?
```

Comment:
@dhh  suggestion.

```
def add_rails_cache
  production_config_file = app_root.join("config/environments/production.rb")

  if production_config_file.exist?
    configure_cache_store(production_config_file)
  end
end

private

def configure_cache_store(config_file)
  gsub_file config_file, /(# )?config\.cache_store = (:.*)/, "config.cache_store = :solid_cache_store"
end
```

Reply:
Got rid of the needlessly defensive guard instead.